lib/utils/telemetry/generatePayload.js

```diff
@@ -108,10 +111,20 @@ module.exports = async (serverless) => {
     return userConfig.get('userId');
   })();
 
+  const isLocallyInstalled = await (async () => {
+    if (serverless) {
+      return serverless.isLocallyInstalled;
+    }
+
+    return Boolean(await resolveLocalServerlessPath());
+  })();
+
+  const { command } = resolveInput();
+
   const payload = {
     ciName,
     cliName: 'serverless',
-    command: serverless.processedInput.commands.join(' '),
+    command,
```

Comment:
Are metrics prepared to accept `""` (empty string) command name ?

Reply:
yes, it will safely accept empty string as command - the only thing I'm wondering about it is if we should default that empty string to some other value? Previously there was an idea for `interactive` as `command`, but I think we can just keep it empty. What do you think?

Reply:
To simplify things (avoid hacking) I'd keep it empty, as empty string it remains easily discoverable in mixed panel, so I think there's no real issue